EIPS/eip-aa.md

```diff
@@ -45,9 +45,14 @@ The following semantics are enforced:
 * AA transactions that do not call `PAYGAS` are considered invalid.
 * After `PAYGAS` is executed, further invokations during the same transaction
   are treated as noops.
-* If `CALLER (0x33)` is invoked in a call intiated by an AA transaction and
```

Comment:
0x7CB5454213b5ffF7e60ad63768d77ea46E621b3A

Reply:
That's a good catch. @SamWilsn, @adietrichs, do you concur?